src/geo/projection/globe_util.js

```diff
@@ -29,6 +29,7 @@ export const GLOBE_RADIUS = EXTENT / Math.PI / 2.0;
 const GLOBE_NORMALIZATION_BIT_RANGE = 15;
 const GLOBE_NORMALIZATION_MASK = (1 << (GLOBE_NORMALIZATION_BIT_RANGE - 1)) - 1;
 const GLOBE_VERTEX_GRID_SIZE = 64;
+const GLOBE_LATITUDINAL_GRID_LOD_TABLE = [GLOBE_VERTEX_GRID_SIZE, GLOBE_VERTEX_GRID_SIZE / 2, GLOBE_VERTEX_GRID_SIZE / 4];
```

Comment:
If I understand this correctly, the memory impact from this change is the addition of 2 buffers (two new LODs + current buffer) instead of the single shared one we have in main?

Reply:
Only the index buffer grows bigger actually. That's because we could still use the same 64x64 vertex buffer to say reference a 32x32 sub-region. This however changes the access pattern slightly.